### DISCUSSION TITLE
icon-request: fa-wood

### DISCUSSION THREAD

**Discussion by leon486** [FortAwesome/Font-Awesome#21199] [Icon Requests]
I would like to request an icon that depicts wood as a material. It will be used for an app that deals with many materials for antiques, wood, clay, stone, bone, etc. It can also be used for games.

I searched and couldn't find a wood icon.

Here are two examples:
[Example 1](https://cdn3.iconfinder.com/data/icons/nature-animals/512/Log_A-512.png)
[Example 2](http://www.thecbg.org/wiki/images/2/23/Wood_Icon.png)

**Comment by Devnsyde** [comment]
I'd like to add that a pile of logs/wood (maybe a campfire without the fire) would also be a nice addition.

**Comment by chakathepakuni** [comment]
+1 log! https://www.youtube.com/watch?v=2C7mNr5WMjA

**Comment by ghost** [comment]
Any update on this?  Having lumber and stacks of lumber would be "awesome"!

**Comment by hinds44** [comment]
Yes!
I would live my “dev logs” button to be a log

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🪵 [U+1FAB5](https://emojipedia.org/emoji/%F0%9F%AA%B5/) Wood

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
